model/artifact/artifact_file.go

```diff
@@ -38,6 +39,9 @@ type File struct {
 	Visibility string `json:"visibility" bson:"visibility"`
 	// When true, these artifacts are excluded from reproduction
 	IgnoreForFetch bool `bson:"fetch_ignore,omitempty" json:"ignore_for_fetch"`
+	//AwsKey and AwsSercret are the key and secret with which the file was uploaded to s3
```

Comment:
Typo AwsSercret. Also the convention in Go (though admittedly older sections of the code base do not follow the convention) is that comments should be immediately above the thing they comment, since that's what godoc expects. To do this move the AwsSecret comment to its own line.